salt/utils/cloud.py

```diff
@@ -935,7 +935,7 @@ def remove_service(self, wait_timeout=10, sleep_wait=1):
 def run_winexe_command(cmd, args, host, username, password, port=445):
     '''
     Run a command remotly via the winexe executable
-    '''
+    '''    
```

Comment:
Pease remove the extra spaces after the doc string. The linter is probably complaining about this one.

Reply:
OK